src/common/Footer/Footer.styled.js

```diff
@@ -0,0 +1,102 @@
+import styled from "styled-components";
+
+export const FooterContainer = styled.footer`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-color: #1b2336;
+  color: ${(p) => p.theme["blue-gray-500"]};
+  padding: 50px;
+
+  .imagens {
+    display: flex;
+  }
+  h2 {
+    color: #${(p) => p.theme["secondary-900"]};
```

Comment:
aqui podemos remover o `#`, pois na definição de tema a cor já vem com o #
```suggestion
    color: ${(p) => p.theme["secondary-900"]};
```